### ISSUE TITLE
Fase 2 -  Priorità?

### ISSUE THREAD

**Issue by iltempe** [emergenzeHack/covid19italia#560]
**Priorità**: media

## Testo della comunicazione
Ciao a tutti secondo voi su cosa è meglio lavorare adesso come priorità? Grazie

**Comment by cristigalas** [comment]
Come ho scritto su Slack, secondo me, anche in previsione del fatto che le segnalazioni stanno diminuendo e credo continueranno a diminuire, sarebbe utile alimentare in modo automatico i nostri **canali Telegram e Twitter** così da tenere comunque viva l’attenzione sul progetto. Propongo quindi di inviare in modo automatico sui nostri canali Telegram e Twitter tutte le **ordinanze su Covid emanate da governo, protezione civile e ministeri** pubblicate in Gazzetta Ufficiale. Sul sito della Gazzetta c’è infatti [questa sezione dedicata](https://www.gazzettaufficiale.it/dettaglioArea/12 ) in continuo aggiornamento, dalla quale credo si possano ricavare i feed con titolo+link da inviare su Telegram e Twitter. Che ne pensate? È fattibile? 
Poi la stessa cosa potremmo farla anche con le **ordinanze e misure adottate dalle Regioni**, dal momento che tutte hanno una sezione web dedicata a Covid. I link alle sezioni Covid delle Regioni li [abbiamo raccolti tutti qui](https://www.covid19italia.help/fonti-istituzionali/#Regionale).
Pensavo cioè ad una sorta di _**Bollettino Normativo**_, che offre un quadro completo e aggiornato dei vari provvedimenti che via via sono adottati a livello nazionale e regionale. Ad oggi non esiste nulla del genere.

**Comment by favoeva** [comment]
Aggiungerei circolari e direttive, 
oltre a doc tecnici varie autorità e istituti e sindacato  e 
Piani d’azione (vedi milano , Torino ) , protocolli, 

Dividerei per temi
Lavoro
Scuola
Pubblica amministrazione 
Imprese e industria
Sanità salute
Mobilità trasporti
Turismo

**Comment by cristigalas** [comment]
> Aggiungerei circolari e direttive,
> oltre a doc tecnici varie autorità e istituti e sindacato e
> Piani d’azione (vedi milano , Torino ) , protocolli,
> 
> Dividerei per temi
> Lavoro
> Scuola
> Pubblica amministrazione
> Imprese e industria
> Sanità salute
> Mobilità trasporti
> Turismo

Queste aggiunte e categorie tematiche le possiamo eventualmente introdurre sul sito nella sezione Fonti istituzionali. Quello che suggerivo io era invece l’invio automatico al Bollettino prendendo i link direttamente dalla sezione della Gazzetta Ufficiale o dai siti delle Regioni. Un po’ come facevamo con Tci per le ordinanze della protezione civile che andavano nel Bollettino direttamente dal sito del Dipartimento.

**Comment by cristigalas** [comment]
Comunque potremmo intanto sentire @aborruso che di Bollettino s’intende.

**Comment by luciaroma** [comment]
concordo su tutto e sicuramente è importante farne un ragionamento e un confronto in call, come già lanciato da @cristigalas in chat.

**Comment by aborruso** [comment]
@cristigalas quindi potrebbe essere utile creare un feed RSS da questa pagina https://www.gazzettaufficiale.it/dettaglioArea/12 ?

Nel gruppo developer pensi che qualcuno se ne possa occupare? Non vorrei fare una cosa, per cui c'è già qualcuno/qualcosa di pronto

**Comment by cristigalas** [comment]
> @cristigalas quindi potrebbe essere utile creare un feed RSS da questa pagina https://www.gazzettaufficiale.it/dettaglioArea/12 ?
> 
> Nel gruppo developer pensi che qualcuno se ne possa occupare? Non vorrei fare una cosa, per cui c'è già qualcuno/qualcosa di pronto

Non credo, il gruppo sviluppatori langue ;) E poi visto che il Bollettino è una tua ‘creatura’ ho pensato a te... Che ne dici? Hai modo e tempo di lavorarci?
Secondo me sarebbe molto utile avere nel Bollettino un aggiornamento automatico di tutti gli atti pubblicati nella sezione Covid-19 della Gazzetta. Come avrai visto al link che ho indicato ci sono varie sottosezioni, una per ogni organo istituzionale, quindi immagino che si debba creare un feed RSS per ogni sottosezione. Sbaglio?

**Comment by chiccap** [comment]
Nella call di riprogrammazione priorità per la Fase 2 segnalo che con @cristigalas pensavamo di portare avanti la discussione su una pagina dati vedi #38. Parliamone insieme!

**Comment by chiccap** [comment]
Pensavo che sotto COME PARTECIPARE in questa fase 2 potremmo mette una cosa tipo "Promuovi anche tu il progetto" con il link al volantino scaricabile.

**Comment by cristigalas** [comment]
Anche in vista della prossima call, ho riepilogato in [questo GgogleDoc](https://docs.google.com/document/d/1Dl7ci-9oTAwznT8xibFg_hSeqGgd1mx1e_3aluDJ3HE/edit) alcune proposte di sviluppo già emerse nelle scorse settimane che credo sia importante tenere presente per la cosiddetta fase 2 del progetto.
